### DISCUSSION TITLE
Torrent clients and network interfaces

### DISCUSSION THREAD

**Discussion by stb9** [qdm12/gluetun#1829] [Q&A]
I have a torrent client (qBit) running in a container using network_mode: service gluetun.

In qBit I have the choice between the following possible network interfaces: eth0, tun0, or "any". ifconfig shows that eth0 in bound to the gluetun docker local IP and tun0 is bound to the wireguard address configured in gluetun.

What is the 'correct' network interface setting in qBit?

**Comment by Lokilicious** [comment]
tun0

if you bind it to eth0 you may circumvent the vpn completely

**Comment by Eduard-Voiculescu** [comment]
This is probably a late comment, but gluetun mentions what Network Interface it uses. In my case, when I look at my logs, this is what I see:  
```bash
Network interface: tun0
```
